new flak8 fails the build

```bash
flake8 tests --isolated
tests/testapp/sixmock.py:2:80: E501 line too long (92 > 79 characters)
make: *** [lint] Error 1
ERROR: InvocationError: '/usr/bin/make test lint'
```

--- a/tests/testapp/sixmock.py
+++ b/tests/testapp/sixmock.py
@@ -1,4 +1,4 @@
 try:
-    from unittest.mock import call, patch, Mock, PropertyMock, MagicMock  # noqa: F401, E501
+    from unittest.mock import call, patch, Mock, PropertyMock, MagicMock  # noqa: E501
 except:
     from mock import call, patch, Mock, PropertyMock, MagicMock  # noqa: F401

